packages/react-chat/src/components/Footer/index.tsx

```diff
@@ -44,8 +49,8 @@ const Footer: React.FC<FooterProps> = ({ withWatermark, hasEnded, onStart, onSen
   };
 
   return (
-    <Container withShadow={!!hasEnded} withWatermark={withWatermark}>
-      {hasEnded ? (
+    <Container withShadow={!!hasEnded || (!autostart && session.status === SessionStatus.IDLE)} withWatermark={withWatermark}>
+      {hasEnded || (!autostart && session.status === SessionStatus.IDLE) ? (
```

Comment:
could we just set `status` to `SessionStatus.ENDED` IF `!autostart`?

This can be done in the runtime state:
https://github.com/voiceflow/react-chat/blob/a3233a6e188504f10c5cf24cb325f8856e965968/packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts#L30